integration/specs/InternalDropdown/internalDropdown-1.spec.js

```diff
@@ -0,0 +1,50 @@
+const PageInternalDropdown = require('../../../src/components/InternalDropdown/pageObject');
+const { ARROW_DOWN_KEY, ARROW_UP_KEY } = require('../../constants');
+
+const INTERNALDROPDOWN = '#internal-dropdown-1';
+
+describe('InternalDropdown base example', () => {
+    beforeAll(() => {
+        browser.url('/#!/InternalDropdown/1');
+    });
+    beforeEach(() => {
+        browser.refresh();
+        const component = $(INTERNALDROPDOWN);
+        component.waitForExist();
+    });
+    it('should render Arrow component at top of the list of options when scrolling down the list', () => {
+        const internalDropdown = new PageInternalDropdown(INTERNALDROPDOWN);
+        internalDropdown.hoverScrollDownArrow();
+        expect(internalDropdown.arrowUpExists()).toBe(true);
+    });
+    it('should not render Arrow component at top of the list of options when scroll is at top of the list', () => {
+        const internalDropdown = new PageInternalDropdown(INTERNALDROPDOWN);
+        internalDropdown.getOption(0);
+        expect(internalDropdown.arrowUpExists()).toBe(false);
+    });
+    it('should render Arrow component at bottom of the list of options when scrolling up the list', () => {
+        const internalDropdown = new PageInternalDropdown(INTERNALDROPDOWN);
+        internalDropdown.getOption(6).click();
+        browser.keys(ARROW_UP_KEY);
+        browser.keys(ARROW_UP_KEY);
+        expect(internalDropdown.arrowDownExists()).toBe(true);
+    });
+    it('should not render Arrow component at bottom of the list of options when last option is visible', () => {
+        const internalDropdown = new PageInternalDropdown(INTERNALDROPDOWN);
+        internalDropdown.getOption(4).click();
+        browser.keys(ARROW_DOWN_KEY);
+        browser.keys(ARROW_DOWN_KEY);
+        browser.keys(ARROW_DOWN_KEY);
+        browser.keys(ARROW_DOWN_KEY);
+        browser.keys(ARROW_DOWN_KEY);
+        expect(internalDropdown.arrowDownExists()).toBe(false);
+    });
+    it('should render Arrow component at top and bottom when first option and last options are not visible', () => {
+        const internalDropdown = new PageInternalDropdown(INTERNALDROPDOWN);
+        const option4 = internalDropdown.getOption(5);
+        option4.hover();
+        browser.keys(ARROW_DOWN_KEY);
+        expect(internalDropdown.arrowUpExists()).toBe(true);
+        expect(internalDropdown.arrowDownExists()).toBe(true);
+    });
```

Comment:
here we should add tests for keyboard navigation, Option page object have an isActive method, then we should test that when press arrow key the next option is active:
- [ ] should set active next option when press arrow down key
- [ ] should select the option clicked
- [ ] should select the option active when press enter key